MSAL/src/public/MSALWebviewParameters.h

```diff
@@ -94,6 +95,13 @@ NS_ASSUME_NONNULL_BEGIN
 */
 - (nonnull instancetype)initWithAuthPresentationViewController:(MSALViewController *)parentViewController;
 
+
+#if !MSID_EXCLUDE_WEBKIT
+
++ (WKWebViewConfiguration *)createWebViewConfigWithPKeyAuthUserAgent;
```

Comment:
Same comment about naming... Let's call this something like "defaultWKWebViewConfiguration"? Also, could we make this a readonly class property perhaps?

Reply:
sounds good!